docs/installation/debian.md

```diff
@@ -12,113 +13,121 @@ parent = "smn_linux"
 
 Docker is supported on the following versions of Debian:
 
+ - [*Debian testing stretch (64-bit)*](#debian-wheezy-stable-7-x-64-bit)
  - [*Debian 8.0 Jessie (64-bit)*](#debian-jessie-80-64-bit)
  - [*Debian 7.7 Wheezy (64-bit)*](#debian-wheezy-stable-7-x-64-bit)
 
-## Debian Jessie 8.0 (64-bit)
+ >**Note**: If you previously installed Docker using `apt`, make sure you update your apt` sources to the new `apt` repository.
 
-Debian 8 comes with a 3.16.0 Linux kernel, the `docker.io` package can be found in the `jessie-backports` repository. Reasoning behind this can be found <a href="https://lists.debian.org/debian-release/2015/03/msg00685.html" target="_blank">here</a>. Instructions how to enable the backports repository can be found <a href="http://backports.debian.org/Instructions/" target="_blank">here</a>.
+ ## Prerequisites
 
-> **Note**:
-> Debian contains a much older KDE3/GNOME2 package called ``docker``, so the
-> package and the executable are called ``docker.io``.
+ Docker requires a 64-bit installation regardless of your Debian version.
+ Additionally, your kernel must be 3.10 at minimum. The latest 3.10 minor
+ version or a newer maintained version are also acceptable.
 
-### Installation
+ Kernels older than 3.10 lack some of the features required to run Docker
+ containers. These older versions are known to have bugs which cause data loss
+ and frequently panic under certain conditions.
 
-Make sure you enabled the `jessie-backports` repository, as stated above.
+ To check your current kernel version, open a terminal and use `uname -r` to
+ display your kernel version:
 
-To install the latest Debian package (may not be the latest Docker release):
+     $ uname -r
 
-    $ sudo apt-get update
-    $ sudo apt-get install docker.io
+### Update your apt repository
 
-To verify that everything has worked as expected:
+Docker's `apt` repository contains Docker 1.7.1 and higher. To set `apt` to use from the new repository:
 
-    $ sudo docker run --rm hello-world
+ 1. If you haven't already done so, log into your Ubuntu instance.
```

Comment:
I think it was this way, but I found "your Ubuntu instance" a strange choice of words, perhaps the same as in the CentOS docs; "Log into your machine as a user with `sudo` or `root` privileges."